modules/apps/frontend-js/frontend-js-spa-web/src/main/resources/META-INF/resources/senna/app/App.js

```diff
@@ -523,7 +521,7 @@ class App extends EventEmitter {
 				this.handleNavigateError_(path, nextScreen, reason);
 				throw reason;
 			})
-			.thenAlways(() => {
+			.finally(() => {
```

Comment:
Yay. I recall @julien wanting to get rid of this back in the day but the need to support IE getting in his way.